_maps/map_files/MetaStation/MetaStation.dmm

```diff
@@ -60089,17 +60089,12 @@
 	},
 /area/station/engineering/control)
 "jnZ" = (
-/obj/machinery/light_switch{
-	dir = 8;
-	name = "east bump";
-	pixel_x = 24
-	},
+/obj/item/kirbyplants/plant24,
+/obj/machinery/atmospherics/unary/vent_scrubber/on,
 /obj/machinery/camera{
-	c_tag = "Detective's Office";
+	c_tag = "Brig Detective's Office";
```

Comment:
brig detective office?

Reply:
That is how box station c_tag named it as a camera in detective office

Reply:
okay only farragus the right C_tag, but the other stations do not have